tests/ci/submitpytest.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+import os
+import json
+import azureml.core
+from azureml.core.authentication import AzureCliAuthentication
+from azureml.core import Workspace
+from azureml.core import Experiment
+from azureml.core.runconfig import RunConfiguration
+from azureml.core.conda_dependencies import CondaDependencies
+from azureml.core.runconfig import DEFAULT_CPU_IMAGE
+# uncomment if using gpu
+from azureml.core.runconfig import DEFAULT_GPU_IMAGE
+from azureml.core.script_run_config import ScriptRunConfig
+
+
+from azureml.core.compute import ComputeTarget, AmlCompute
+from azureml.core.compute_target import ComputeTargetException
+
+def submit_exp():
+    with open("tests/ci/config.json") as f:
+            config = json.load(f)
+
+            workspace_name= config["workspace_name"]
+            resource_group = config["resource_group"]
+            subscription_id = config["subscription_id"]
+            location = config["location"]
+
+            print(" WS name ", workspace_name)
+            print("subscription_id ", subscription_id)
+            print("location",location)
+
+            cli_auth = AzureCliAuthentication()
+            print("cliauth")
+
+    try:
+        print("Trying to get ws")
```

Comment:
in other parts of the code we are using logs instead of prints, here there is an example: https://github.com/microsoft/recommenders/blob/master/reco_utils/recommender/sar/sar_singlenode.py#L20

Reply:
addeded logging.